08-AsyncIO/AsyncIO/Tasks.cs

```diff
@@ -53,7 +94,19 @@ public static IEnumerable<string> GetUrlContentAsync(this IEnumerable<Uri> uris,
         public static Task<string> GetMD5Async(this Uri resource)
         {
             // TODO : Implement GetMD5Async
-            throw new NotImplementedException();
+            //throw new NotImplementedException();
+
+            return  Task.Run<string>(() =>
```

Comment:
don't create new task, the api is already asynchronous use ContinueWith method or async/await feature